### PR TITLE
Read Excel Util

### PR DESCRIPTION
**Pull request by Subha6636**
Today I have added Read Excel Util. Please Review.

### CODE REVIEW

**Comment by bikash9090** [comment]
Hi @Subha6636 i just reviewed the code and found some code which need to change. below are the points need to change.
1. Set the path of excel file as dynamic.
2. Give some appropriate message in catch block when the exception thrown.
3. Make the excel sheet reading reusable by parameterizing the readExcel() with sheet name.

Thank You!

File: src/test/java/com/taspro/utility/ExcelUtil.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+public class ExcelUtil {
+	
+	XSSFWorkbook workbook;
+	
+	
+	public String[][] readExcel() {
+	String path = "C:\\Users\\Subha Sangita Swain\\git\\tas-pro\\src\\test\\resources\\TAS-2_Employee_data.xlsx";
```
Comment (bikash9090) [inline src/test/java/com/taspro/utility/ExcelUtil.java:16]: Make it dynamic path allocation during runtime.

**Review by bikash9090 — CHANGES_REQUESTED** [review]
Hi @Subha6636  i just reviewed the code and found some code which need to change. below are the points mentioned.

1. Set the path of excel file as dynamic usng System.getProperty("user.dir").
2. Give some appropriate message in catch block when the exception thrown.
3. Make the excel sheet reading reusable by parameterizing the readExcel() with sheet name.

Thank You!

**Comment by Subha6636** [comment]
I have updated. Please review and merge.

**Review by bikash9090 — APPROVED** [review]
Yes Subha Sangita it is alright, go ahead and merge to master.

**Pull request merged**